authentication/serializers.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth import update_session_auth_hash
+
+from rest_framework import serializers
+
+from .models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(write_only=True, required=True)
+
+    class Meta:
+        model = User
+        fields = (
+            'id', 'email', 'username',
+            'first_name', 'last_name', 'password')
+
+    def create(self, validated_data):
+        return User.objects.create_user(**validated_data)
+
+

```

Comment:
@ralucamiclea, @davidluca, So what happens when we update the user and also send the password ? I remember you've overridden `update` method exactly for this reason.